docs/src/multivariate.md

```diff
@@ -31,8 +31,8 @@ entropy(::MultivariateDistribution, ::Real)
 
 ```@docs
 insupport(::MultivariateDistribution, ::AbstractArray)
-pdf(::MultivariateDistribution, ::AbstractArray)
-logpdf(::MultivariateDistribution, ::AbstractArray)
+pdf(::Distribution{ArrayLikeVariate{N}}, x::AbstractArray{<:Real,M}) where {N,M}
+logpdf(::Distribution{ArrayLikeVariate{N}}, x::AbstractArray{<:Real,M}) where {N,M}
```

Comment:
I'm not sure... This page is about `MultivariateDistribution` but the methods listed here are for generic distributions with array-like variates. I think it would be better to include these generic docstrings in a separate page and only list here `pdf`, `logpdf` etc. as part of the interface for multivariate distributions with a link to the generic docstring.

Reply:
That's a valid concern, I was just following the docstring for `loglikelihood` for array-like variates which is currently listed on the multivariate page, so I thought it would be consistent. https://github.com/JuliaStats/Distributions.jl/blob/f33af97da34a2e772837b63891c0ed52a3a1a989/src/common.jl#L433-L443
If you think these should be taken to a separate page, then `loglikelihood` probably should as well